### PR TITLE
 Add withdraw and Deposit claimable logic

### PR DESCRIPTION
**Pull request by morgueye4**

### CODE REVIEW

File: src/lib/config.ts [hunk 1]
```diff
@@ -3,17 +3,19 @@ const config = {
...
   address_type: 42, // https://github.com/paritytech/substrate/blob/e232d78dd5bafa3bbaae9ac9db08f99e238392db/primitives/core/src/crypto.rs#L444
-  mnemonic: process.env.REACT_APP_FAUCET_MNEMONIC_SEED,
+  mnemonic: "0xabf8e5bdbe30c65656c0a3cbd181ff8a56294a69dfedd27982aace4a76909115",//process.env.REACT_APP_FAUCET_MNEMONIC_SEED,
   amount: 10000,
   limit: 2, // The time limit for sending requests is in hours.,
   friend_bot_url: "https://friendbot.stellar.org?addr=",
   horizon_testnet_url: "https://horizon-testnet.stellar.org",
   issuer_public: "GAKNDFRRWA3RPWNLTI3G4EBSD3RGNZZOY5WKWYMQ6CQTG3KIEKPYWAYC",
-  issuer_secret: process.env.REACT_APP_ASSET_ISSUER_SECRET || "",
+  issuer_secret: "SA4OOLVVZV2W7XAKFXUEKLMQ6Y2W5JBENHO5LP6W6BCPBU3WUZ5EBT7K",//process.env.REACT_APP_ASSET_ISSUER_SECRET || "",
```
Comment (gonzamontiel) [inline src/lib/config.ts]: Please remove the secret.

File: src/lib/api.ts [hunk 2]
```diff
@@ -2,177 +2,233 @@ import { ApiPromise, WsProvider } from "@polkadot/api";
 import uiKeyring from "@polkadot/ui-keyring";
```
Comment (gonzamontiel) [inline src/lib/api.ts]: Please run eslint on all the files so you don't commit style changes (in this case the incoming change have a tab of 4 spaces, while the linter has 2 spaces)

**Review by gonzamontiel — CHANGES_REQUESTED** [review]

File: src/lib/config.ts (hunk 1 above)
Comment (morgueye4) [reply src/lib/config.ts]: it is not actually pushed it's an outdated one.

Comment (gonzamontiel) [reply src/lib/config.ts]: You silently removed it in a merge strategy https://github.com/pendulum-chain/playground/pull/26/commits/16c4e73c570e2f8fbb658c954c661cf9577658f1
Please, try to be more explicit about this, ideally with atomic and small commits. explaining the changes.

**Review by gonzamontiel — APPROVED** [review]
Ready to merge!

**Pull request merged**